store/stellar.go

```diff
@@ -7,17 +7,63 @@ import (
 	sq "github.com/Masterminds/squirrel"
 )
 
+type StellarDeposit struct {
+	// ID is the globally unique id for this deposit
+	ID string `db:"id"`
+	// Asset is the string encoding of the Stellar assets
+	// which were deposited to the bridge
+	Asset string `db:"token"`
+	// Sender is the address of the account which deposited the tokens
+	Sender string `db:"sender"`
+	// Destination is the intended recipient of the bridge transfer
+	Destination string `db:"destination"`
+	// Amount is the amount of tokens which were deposited to the bridge
+	// contract
+	Amount string `db:"amount"`
+	// LedgerTime is the unix timestamp of the deposit
+	LedgerTime int64 `db:"ledger_time"`
```

Comment:
Why not `time.Time`?

Reply:
I had some issues storing `time.Time` values in the db in the integration tests which is why I decided to just store the unix timestamp instead

Reply:
I used it starting at https://github.com/stellar/starbridge/tree/d0314d3e51b7205ec8aaae278825765dc943d46c for `outgoing_stellar_transactions.expiration`. The type was `timestamp without time zone` and it was working fine.